server/app/views/admin/programs/ProgramTranslationView.java

```diff
@@ -176,4 +155,46 @@ private ImmutableList<DomContent> formFields(
     }
     return result.build();
   }
+
+  private ImmutableList<DomContent> getApplicantVisibleProgramDetailFields(
+      ProgramDefinition program, LocalizationUpdate updateData) {
+    ImmutableList.Builder<DomContent> applicantVisibleDetails =
+        ImmutableList.<DomContent>builder()
+            .add(
+                fieldWithDefaultLocaleTextHint(
+                    FieldWithLabel.input()
+                        .setFieldName(ProgramTranslationForm.DISPLAY_NAME_FORM_NAME)
+                        .setLabelText("Program name")
+                        .setValue(updateData.localizedDisplayName())
+                        .getInputTag(),
+                    program.localizedName()),
+                fieldWithDefaultLocaleTextHint(
+                    FieldWithLabel.input()
+                        .setFieldName(ProgramTranslationForm.DISPLAY_DESCRIPTION_FORM_NAME)
+                        .setLabelText("Program description")
+                        .setValue(updateData.localizedDisplayDescription())
+                        .getInputTag(),
+                    program.localizedDescription()),
+                fieldWithDefaultLocaleTextHint(
+                    FieldWithLabel.input()
+                        .setFieldName(ProgramTranslationForm.CUSTOM_CONFIRMATION_MESSAGE_FORM_NAME)
+                        .setLabelText("Custom Confirmation Screen Message")
+                        .setValue(updateData.localizedConfirmationMessage())
+                        .getInputTag(),
+                    program.localizedConfirmationMessage()));
+
+    // Only add the summary image description input to the page if a summary image description is
```

Comment:
praise: I really like this design choice! It would be nice to have this conditional rendering with the confirmation message too.

Reply:
Thank you! Let me know if you want me to file a GH issue about it.

Reply:
That would be great! Thanks!